opentreemap/works_management/tests.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+from django.contrib.gis.geos import Point
+from django.utils import timezone
+
+from treemap.models import Plot
+from treemap.tests.base import OTMTestCase
+from treemap.tests import make_instance, make_commander_user
+
+from works_management.models import Team, Task, WorkOrder
+
+
+def make_team(instance, name='Test Team'):
+    t = Team()
+    t.instance = instance
+    t.name = name
+    t.save()
+    return t
+
+
+def make_work_order(instance, user):
+    w = WorkOrder()
+    w.instance = instance
+    w.created_by = user
+    w.save()
+    return w
+
+
+def make_task(instance, user, map_feature, name='Test Task', team=None):
+    t = Task()
+    t.instance = instance
+    t.map_feature = map_feature
+    t.team = team
+    t.name = name
+    t.created_by = user
+    t.requested_on = timezone.now()
+    t.scheduled_on = timezone.now()
+    t.closed_on = timezone.now()
+    t.save_with_user(user)
+    return t
+
+
+def make_plot(instance, user):
+    p = Point(0, 0)
+    plot = Plot(geom=p, instance=instance)
+    plot.save_with_user(user)
+    return plot
+
+
+class WorksManagementTests(OTMTestCase):
```

Comment:
I would like to add these two tests.

```python
    def test_cannot_call_save_on_workorder(self):
        # WorkOrder is Auditible and should only allow calling save_with_user.
        w = make_work_order(self.instance, self.user)
        with self.assertRaises(UserTrackingException):
            w.save()

    def test_cannot_call_save_on_task(self):
        # Task is Auditible and should only allow calling save_with_user.
        t = make_task(self.instance, self.user,
                      make_plot(self.instance, self.user))
        with self.assertRaises(UserTrackingException):
            t.save()
```

The WorkOrder test fails. I would expect `save` function in Auditible to be called, but it is not. 

https://github.com/OpenTreeMap/otm-core/blob/746322311ef4df3e550dfdcfd81f9848a5202c53/opentreemap/treemap/audit.py#L615-L618

If we cannot figure out the inheritance issue, the simple solution is to copy the `save` function into the WorkOrder model.

```
    def save(self, *args, **kwargs):
        raise UserTrackingException(
            'All changes to %s objects must be saved via "save_with_user"' %
            (self._model_name))
```